readthedocs/projects/tests/test_views.py

```diff
@@ -270,3 +270,25 @@ def test_download_files(self):
                 resp["X-Accel-Redirect"],
                 f"/proxito/media/{type_}/project/latest/project.{extension}",
             )
+
+
+class TestProjectAdvancedUpdate(TestCase):
+    def setUp(self):
+        self.user = get(User)
+        get(EmailAddress, email=self.user.email, user=self.user, verified=True)
+        self.project = get(Project, users=[self.user])
+
+    def test_project_advanced_update_with_valid_simple_form_data(self):
+        url = reverse("projects_advanced", args=[self.project.slug])
+        self.client.force_login(self.user)
+        resp = self.client.post(
+            url,
+            data={
+                "default_version": "latest",
+                "default_branch": "",
+                "documentation_type": "sphinx",
+                "python_interpreter": "python3",
+                "build_config_file": "test.yaml",
+            },
+        )
+        self.assertEqual(resp.status_code, 302)
```

Comment:
That's a good addition to the test. I didn't wanna write a heap of stuff since I'm testing something that had 0 testing before, so I consider it already a huge improvement :)

Reply:
Added :+1: